sites/fast-color-explorer/app/components/gradient/gradient.template.ts

```diff
@@ -0,0 +1,31 @@
+import { parseColor } from "@microsoft/fast-colors";
+import { isDark, SwatchRGB } from "@microsoft/fast-components";
+import { html, repeat } from "@microsoft/fast-element";
+import { Gradient } from "./gradient";
+
+function getColor(background) {
+    const bg = parseColor(background);
+    const darkMode = isDark(SwatchRGB.from(bg));
+    return darkMode ? "white" : "black";
+}
+
+export const gradientTemplate = html<Gradient>`
+    <template>
```

Comment:
Just FYI: It doesn't hurt to have these `<template>` tags here, but you only need them when binding to data (generally attributes or event listeners) to the element host. Otherwise, you can safely omit it.

Reply:
I've wondered about this. So essentially `<template>` by itself is not necessary, but `<template class="${x => x.something}">` is an example of how it could be used?